app/index.html

```diff
@@ -58,6 +58,10 @@ <h6 class="docs-header">What is this for?</h6>
             <label for="movie-data">MOVIES Slot</label>
             <textarea class="u-full-width text-area" id="movie-data" disabled></textarea>
           </div>
+          <div class="six columns">
+            <label for="moviegenre-data">MOVIEGENRES Slot</label>
+            <textarea class="u-full-width text-area" id="moviegenre-data" disabled></textarea>
+          </div>
         </div>
```

Comment:
I also spotted this and submitted a PR to @jagsta's repo altering it (not sure what the correct etiquette was)